src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -90,7 +90,9 @@ export const TooltipComponent: StoryFn<TooltipProps> = (args: TooltipProps) => {
                 {...args}
                 triggerElement={
                     <button className="tw-flex tw-justify-center tw-items-center">
-                        <IconExclamationMarkCircle size={IconSize.Size16} filled />
+                        <span className="tw-flex tw-leading-3">
+                            <IconExclamationMarkCircle size={IconSize.Size16} filled />
```

Comment:
Here you can use the `IconExclamationMarkCircle16Filled` component directly, as we know already the size we want to use. We'd like that to be the norm and encourage people to use the sized Icon component whenever possible.